shard.yml

```diff
@@ -3,11 +3,14 @@ name: amber
 version: 0.6.4
 
 authors:
-  - Elias Perez <<eliasjpr@gmail.com>>
-  - Isaac Sloan <<isaac@isaacsloan.com>>
-  - Dru Jensen <<drujensen@gmail.com>>
-  - Nick Franken <<shnick@gmail.com>>
-  - Benoit de Chezelles <<benoit.dechezelles@gmail.com>>
+  - Benoit de Chezelles <benoit.dechezelles@gmail.com>
+  - Dru Jensen <drujensen@gmail.com>
+  - Elias Perez <eliasjpr@gmail.com>
+  - Faustino Aguilar <faustinoaq@gmail.com>
+  - Isaac Sloan <isaac@isaacsloan.com>
+  - Mark Siemers <mark.siemers@gmail.com>
+  - Nick Franken <shnick@gmail.com>
+  - Robert Carpenter <>
```

Comment:
@robacarp What''s your email?

Reply:
@robacarp is in the list of maintainers for your convenience `robacarp](https://github.com/robacarp) Robert Carpenter - Maintainer`